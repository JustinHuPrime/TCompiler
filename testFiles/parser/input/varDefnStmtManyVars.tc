module foo;

void bar() {
  int i, j, k;
}