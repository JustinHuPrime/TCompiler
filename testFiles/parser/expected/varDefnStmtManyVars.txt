testFiles/parser/varDefnStmtManyVars.tc (code):
FILE(1, 1, STAB(ENTRY(bar, FUNCTION(testFiles/parser/varDefnStmtManyVars.tc, 3, 1, void()))), MODULE(1, 1, ID(1, 8, foo, REFERENCES())), FUNDEFN(3, 1, KEYWORDTYPE(3, 1, void), ID(3, 6, bar, REFERENCES(testFiles/parser/varDefnStmtManyVars.tc, 3, 1)), STAB(), COMPOUNDSTMT(3, 12, STAB(ENTRY(i, VARIABLE(testFiles/parser/varDefnStmtManyVars.tc, 4, 7, int)), ENTRY(k, VARIABLE(testFiles/parser/varDefnStmtManyVars.tc, 4, 13, int)), ENTRY(j, VARIABLE(testFiles/parser/varDefnStmtManyVars.tc, 4, 10, int))), VARDEFNSTMT(4, 3, KEYWORDTYPE(4, 3, int), ID(4, 7, i, REFERENCES(testFiles/parser/varDefnStmtManyVars.tc, 4, 7)), ID(4, 10, j, REFERENCES(testFiles/parser/varDefnStmtManyVars.tc, 4, 10)), ID(4, 13, k, REFERENCES(testFiles/parser/varDefnStmtManyVars.tc, 4, 13)), (null), (null)))))
